RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -23,6 +23,9 @@
 from RecoJets.JetProducers.fixedGridRhoProducerFastjet_cfi import *
 fixedGridRhoFastjetAllTmp = fixedGridRhoFastjetAll.clone(pfCandidatesTag = cms.InputTag("particleFlowTmp"))
 
+from RecoJets.JetProducers.ak4PFJets_cfi import *
+ak4PFJetsTmp = ak4PFJets.clone(src = cms.InputTag("particleFlowTmp"))
```

Comment:
these are "Calo jets for tracks", not track jets.
The track jets used in the sorting algo are not saved